mineral-ui/mineral-ui
# [Website] Refactor DemoLayout

### Description
* Add shared component `DemoLayout` and use where appropriate in site demos

### Motivation and context
* Reduces much repetition among site demos

### Screenshots, videos, or demo, if appropriate

https://shared-demo-layout--mineral-ui.netlify.com/

### How to test

1. Navigate to Avatar, Button, Checkbox, FormField, Link, Radio, TextArea & TextInput and ensure no changes

### Types of changes
- Refactor

### Checklist
